proto/osmosis/superfluid/superfluid.proto

```diff
@@ -25,17 +25,10 @@ message SuperfluidAsset {
   SuperfluidAssetType asset_type = 2;
 }
 
-// SuperfluidAssetInfo stores the information of superfluid asset - real time changes
-message SuperfluidAssetInfo {
+// SuperfluidIntermediaryAccount takes the role of intermediary between LP token and OSMO tokens for superfluid staking
+message SuperfluidIntermediaryAccount {
   string denom = 1;
-  string total_staked_amount = 2 [ 
-    (gogoproto.moretags) = "yaml:\"total_staked_amount\"",
-    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
-    (gogoproto.nullable) = false
-  ];
```

Comment:
Should we keep this, or is it put in state somewhere? Seems useful to know, though I guess we could push that to an off-chain db

Reply:
Total staked amount can be queried by queries - I didn't find any use case for total staked amount to be stored on chain and removed for simplificaiton.